x-pack/legacy/plugins/ml/public/overview/components/anomaly_detection_panel/anomaly_detection_panel.tsx

```diff
@@ -31,7 +31,7 @@ export interface Group {
   docs_processed: number;
   earliest_timestamp: number;
   latest_timestamp: number;
-  max_anomaly_score: number | null;
+  max_anomaly_score: number | undefined;
```

Comment:
You've removed `null` from type but the comment below tells `maxScore will be null if it was not loaded correctly`

Reply:
Yeah this is the initial value when the component is first initialized. `undefined` to indicate it hasn't been loaded yet so we can know when to show the loading icon and then `null` once it's tried to be loaded but has failed.